__tests__/fixtures/portman.crm.json

```diff
@@ -54,9 +54,9 @@
     ],
     "extendTests": [
       {
-        "openApiOperationId": "companiesAll",
```

Comment:
should we not foresee 2 fixtures to validate both cases?

Reply:
well the rest of the fixture also has a mix of openApiOperationId, and openApiOperation... which means adding this we have all three covered